ink-wrapper-types/Cargo.toml

```diff
@@ -22,12 +22,14 @@ anyhow = { version = "1.0.51" }
 thiserror = { version = "1.0.40" }
 
 # drink specific integration.
-drink = { version = "0.8", optional = true }
+drink = { version = "0.8.7", optional = true }
+drink-test-macro = { version = "0.8.7", optional = true }
 ink_metadata = { version = "4.3.0", optional = true }
+pallet-contracts-primitives = { version = "26.0.0", optional = true }
```

Comment:
Is this something new in drink? i.e.

Reply:
no, `StorageDeposit` is not exposed from either `drink` nor `pallet_contracts` and I didn't want to double declare it, since we are depending on this crate anyway